presto-main/src/main/java/io/prestosql/operator/scalar/MathFunctions.java

```diff
@@ -607,6 +607,42 @@ public static long random(@SqlType(StandardTypes.BIGINT) long value)
         return ThreadLocalRandom.current().nextLong(value);
     }
 
+    @Description("a pseudo-random number between m and n (exclusive)")
```

Comment:
What do you think of saying `between start and stop (exclusive)` here instead of `between m and n (exclusive)` here? That would make it consistent with the error message.

Reply:
Great idea! Thanks for the advice! 
I have a question here. Should I include a `checkCondition` to negative values?

Reply:
I think negative numbers are valid here. Please add some tests to cover, e.g., -10 to -5 and -10 to 10.